HW-6

pekarskaya opened the pull request:
game #2 &tasks of lesson 6

pekarskaya committed:
HW-6
game #2 &tasks of lesson 6

squall2229 commented on hw-6-t1-15.js:
```diff
@@ -0,0 +1,136 @@
+
+const array = [1, 5, 4, 10, 0, 3];
+for (let i = 0; i < array.length; i++) {
+    if(array[i] === 0) 
```
Задание решено не верно, мы не должны ориентироваться на числа после или до искомого числа, массив в теории может быть разный, и число 10 должно быть найдено в любых ситуациях

squall2229 replied:
выполнено

squall2229 commented on hw-6-t1-15.js:
```diff
@@ -0,0 +1,136 @@
+//Задание 4/ С помощью вложенных циклов создайте многомерный массив вида: [[1, 1, 1], [1, 1, 1], [1, 1, 1]]
+
+for (let row = 0; row < 3; row++) {
+	arr[row] = [1, 1, 1]; 
```
задание решено не совсем верно. Нужно сгенерировать массив начиная с исходного пустого массива, то есть заполнить пустой массив тремя массивами, и каждый из под-массивов заполнить еденичками

squall2229 replied:
и задание нужно решить за цикл + вложенный цикл

squall2229 replied:
выполнено

squall2229 commented on hw-6-t1-15.js:
```diff
@@ -0,0 +1,136 @@
+ allElements.sort();
+ console.log(allElements);
+
+ let onlyNumbers = allElements.filter(allElement => allElement !=='a');
```
вместо filter проще взять метод pop

pekarskaya replied:
Я так сначала и сделала - логично же после sort. А потом заглянула в подсказку вашу. А там фильтр. Его можно же и без sort. Ну, вероятно, специально придумали задачку, чтобы жизнь раем не казалась.

squall2229 replied:
вот эта строчка написана не верно - allElements2.sort() * allElements2.pop();
Тут не нужен знак умножения

squall2229 commented on hw-6-t1-15.js:
```diff
@@ -0,0 +1,136 @@
+
+
+ //Задание 7.Дан массив: [9, 8, 7, 6, 5].Попросите пользователя угадать число с помощью метода prompt. Если значение, которое ввел пользователь, есть в массиве, выведите в alert «Угадал», в противном случае — «Не угадал».
+do{
```
давай воспользуемся циклом for или forEach, цикл do/while почти не используется в разработке, предлагаю сразу учиться применять актуальные методы

pekarskaya replied:
Этого в задании вообще нет. do...while здесь было для повторения цикла при желании учасника. В задании этого нет, я убрала

pekarskaya committed:
modified:   hw-6-t1-15.js

pekarskaya committed:
new file:   hw-5-gameGuess1.js
modified:   hw-5.js
deleted:    hw-5task1.js
	deleted:    hw-5task2.js
	deleted:    hw-5task3.js
	deleted:    hw-5task4.js
	deleted:    hw-5task5.js
	deleted:    hw-5task6.js
	deleted:    hw-5task7circle.js
	modified:   hw-6-simpleMath.js
	modified:   hw-6-t1-15.js
	modified:   index.html
